public/robots.txt

```diff
@@ -0,0 +1,14 @@
+# Allow all crawlers
+User-agent: *
+Allow: /
+
+# Sitemaps
+Sitemap: https://tscircuit.com/sitemap.xml
+
+# Disallow specific paths
+Disallow: /api/
+Disallow: /dev-login
+Disallow: /authorize
```

Comment:
haha we don't really have any of these routes, maybe comment it out for now? Just to avoid confusion